src/components/profile/user-profile.js

```diff
@@ -103,11 +105,9 @@ class UserItemView extends Component {
                     TransitionComponent={Zoom}
                   >
                     <IconButton
-                      className={attitude === "0" ? "text-success" : ""}
+                      className={attitude === 0 ? "text-success" : ""}
```

Comment:
`className={!attitude ? "text-success"}`

Reply:
Change only in first example, if change in another examinations like and dislike don`t work correct